### PR TITLE
Switch to using `hmpps-monitoring` library

### PR DESCRIPTION
**Pull request by Thomas-Geraghty**
- Remove existing health check functions
- Switch to using new `hmpps-monitoring` library for performing more robust health checks

### CODE REVIEW

**Comment by alexw-moj** [comment]
Is this going to cause our pods to cycle (probably for no reason) if one or more of its dependencies reports unhealthy?

File: server/config.ts [hunk 1]
```diff
@@ -122,6 +118,7 @@ export default {
     },
     coordinatorApi: {
       url: get('COORDINATOR_API_URL', 'https://arns-coordinator-api-dev.hmpps.service.justice.gov.uk'),
+      healthPath: '/health/ping',
```
Comment (alexw-moj) [inline server/config.ts:121]: Turned all of the above off in docker, but this one never reports DOWN...

Comment (Thomas-Geraghty) [reply server/config.ts:121]: Did you turn off the wiremock docker container too?

Comment (Thomas-Geraghty) [inline server/config.ts:121]: Apologies @alexw-moj my GitHub just messed up and deleted your comment. For future reference, it was a question along the lines of "I've disabled all the containers but this health check continued to pass".

My follow up comment was; did you ensure to stop the Wiremock container too? On our local development environment, it uses the wiremock for the coordinator.

**Comment by Thomas-Geraghty** [comment]
> Is this going to cause our pods to cycle (probably for no reason) if one or more of its dependencies reports unhealthy?

Likely, but this is probably a wider issue and not related specifically to introducing this library

File: server/config.ts (hunk 1 above)
Comment (alexw-moj) [reply server/config.ts:121]: Lol, no I deleted it just as you posted, when I figured that out myself :)

**Comment by alexw-moj** [comment]
> > Is this going to cause our pods to cycle (probably for no reason) if one or more of its dependencies reports unhealthy?
> 
> Likely, but this is probably a wider issue and not related specifically to introducing this library

Does the `isEnabled` option disable the healthcheck itself, or just whether it's used in the calculation of `UP` or `DOWN`? Looks like it just turns them off, but the calculation thing might be useful?

**Review by alexw-moj — APPROVED** [review]

**Comment by Thomas-Geraghty** [comment]
>Does the isEnabled option disable the healthcheck itself, or just whether it's used in the calculation of UP or DOWN? Looks like it just turns them off, but the calculation thing might be useful?

Yep it does currently just outright turn them off. The logic being that you could enable/disable certain healthchecks programatically based on some kind of state/environment through a boolean

**Pull request merged**